notebooks/analyze_looping_across_lending_protocols.ipynb

```diff
@@ -99,9 +99,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import pandas as pd\n",
     "import mysql.connector\n",
-    "from IPython.display import display  # Only needed for Jupyter Notebook\n",
     "\n",
     "# Connect to the MySQL database\n",
```

Comment:
Pls have this cell and the one above commented since we don't use them atm.

Reply:
Done! ✅